README.md

```diff
@@ -295,6 +295,8 @@ Parameter | Description | Default
 `resources.frontend.limits.cpu` | CPU resource limit for the frontend | `2`
 `resources.frontend.limits.memory` | Memory resource limit for the frontend | `2Gi`
  |  | 
+`neoload.configuration.externalTlsTermination` | Must be set to true if TLS termination is [handled by your own way](#external-tls-termination).  | `false`
```

Comment:
Rephrase : 
* Must be set to true if TLS termination is handled by a component outside of the Helm Chart management